library/Middleware/NotFoundMiddleware.php

```diff
@@ -8,7 +8,7 @@
 use Niden\Traits\ResponseTrait;
 use Phalcon\Mvc\Micro;
 use Phalcon\Mvc\Micro\MiddlewareInterface;
-use Phalcon\Mvc\User\Plugin;
+use Phalcon\Plugin;
```

Comment:
@Jeckerson We're going to remove this class in final 4.0.0. Could you use Injectable instead?
See: https://github.com/phalcon/cphalcon/pull/14360

cc @niden

Reply:
Specified as action point in #23